x-pack/plugin/sql/sql-action/src/test/java/org/elasticsearch/xpack/sql/action/TestSqlClearCursorRequest.java

```diff
@@ -33,7 +33,8 @@ public TestSqlClearCursorRequest(RequestInfo requestInfo, String cursor) {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         org.elasticsearch.xpack.sql.proto.SqlClearCursorRequest protoInstance = new org.elasticsearch.xpack.sql.proto.SqlClearCursorRequest(
             this.getCursor(),
-            this.requestInfo()
+            this.requestInfo(),
+            false
```

Comment:
Do the tests require an always `false` value, or could it be random boolean?

Reply:
That was my first implementation, I had to revert it since the tests create multiple instances of this stub request and then compare them; if they are different (ie. randomised), the tests fail. A refactor is a bit complex here, and would impact multiple completely unrelated test cases